solr/test-framework/src/java/org/apache/solr/SolrTestCaseHS.java

```diff
@@ -190,9 +190,23 @@ public static String getQueryResponse(String wt, SolrParams params) throws Excep
     p.remove("qt");
     p.set("indent", "true");
 
-    DirectSolrConnection connection = new DirectSolrConnection(h.getCore());
-    String raw = connection.request(path, p, null);
-    return raw;
+    try (EmbeddedSolrServer server =
+        new EmbeddedSolrServer(h.getCoreContainer(), h.getCore().getName())) {
+      QueryRequest query = new QueryRequest(p);
+      if (path != null) {
+        query.setPath(path);
+      }
```

Comment:
You could remove these 3 lines for `path`, and also remove the 2 lines close above that remove `qt` from params.  EmbeddedSolrServer will resolve `qt` indirectly.